static/css/perdiem.css

```diff
@@ -4,7 +4,7 @@ h1, h2, h3, h4, h5, h6, p, li, input [type=submit] {
 }
 
 h4 [type=submit] {
-    font-family: 'Roboto', sans-serif;
+    font-family: 'roboto', sans-serif;
```

Comment:
Oh okay, that's super weird. Have you tried loading the fonts from base.html as lowercase?